### PR TITLE
fix(components): Fix form field inconsistent spacing when used under Form and InputGroup JOB-37646

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

The `<Input* />` components aren't contained in a single tag. In some cases, those are okay but the input's also renders an error and now description below the field, thus opening those elements to be updated by anything that encapsulates it such as `<Form />` and `<InputGroup />`.

### Standalone input 

image

### In a `<Form />`

image

### In an `<InputGroup />`

image

### Added

- 

### Changed

- 

### Deprecated

- 

### Removed

- 

#### Standalone input 

image

#### In a `<Form />`

image

#### In an `<InputGroup />`

image

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/FormField/FormFieldWrapper.tsx [hunk 1]
```diff
@@ -100,14 +104,18 @@ export function FormFieldWrapper({
...
+      {!inline && (description || error) && (
+        <div>
+          {description && (
+            <FormFieldDescription
+              id={descriptionIdentifier}
+              description={description}
+            />
+          )}
+          {error && <InputValidation message={error} />}
+        </div>
```
Comment (robbyjobber) [inline packages/components/src/FormField/FormFieldWrapper.tsx:116]: I like this so much better

**Comment by robbyjobber** [comment]
@darryltec Do we want to address the spacing on this?  Seems like the error/description is getting cut off 

image
 
image

**Review by robbyjobber — CHANGES_REQUESTED** [review]
Codewise LGTM. looks like some CSS changes needed tho 📓

**Comment by darryltec** [comment]
> Do we want to address the spacing on this? Seems like the error/description is getting cut off

@robbyjobber on the inputgroup? No, that one is a lot harder

**Review by robbyjobber — APPROVED** [review]
> @robbyjobber on the inputgroup? No, that one is a lot harder
ah I see.  In that case LGTM then

**Pull request merged**